stroeer/page/article/v1/article_page.proto

```diff
@@ -44,15 +44,9 @@ message RelatedArticle {
   // thus not containing any data that is only required on detail pages).
   stroeer.core.v1.Article article = 1;
 
-  // Source of the related article.
-  RelatedArticleSource source = 2;
-}
+  // Deprecated source of the related article in form of an enum.
+  reserved 2;
 
-// Source of the related article.
-enum RelatedArticleSource {
-  // Not specified.
-  RELATED_ARTICLE_SOURCE_UNSPECIFIED = 0;
-
-  // Article was related manually by the editorial department.
-  RELATED_ARTICLE_SOURCE_EDITORIAL = 1;
+  // Source of the related article.
+  string related_article_source = 3;
```

Comment:
we don't need an intermediate version with both the enum and the new string field? If that's the case, we could also name the field `source`?!